pkg/unikontainers/types/types.go

```diff
@@ -91,6 +91,9 @@ type ExecArgs struct {
 	VCPUs         uint     // The number of vCPUs to allocate
 	UnikernelPath string   // The path of the unikernel inside rootfs
 	InitrdPath    string   // The path to the initrd of the unikernel
+	VAccelType    string   // The vAccel type {"virtio", "vsock", ""}
+	VSockDevPath  string   // The unix socket path to be bind mounted for fc 
```

Comment:
I think the comment here is a bit misleading. If we construct these arguments then the socket should already have been bind mounted.